### PR TITLE
Embeddings finetuning

### PR DESCRIPTION
**Pull request by strickvl**

### CODE REVIEW

File: llm-complete-guide/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-zenml
+zenml[server]==0.58.2
```
Comment (strickvl) [inline llm-complete-guide/requirements.txt:1]:
```suggestion
zenml[server]>=0.63.0
```

File: llm-complete-guide/pipelines/finetune_embeddings_legacy.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        train_dataset,
+        model_path=MODEL_PATH,
+        num_epochs=NUM_EPOCHS,
+        warmup_steps=WARMUP_STEPS,
+    )
+
+    evaluate_model(model, MODEL_PATH, test_dataset)
+
+
+# # TODO: FOR TESTING ONLY (remove after)
```
Comment (AlexejPenner) [inline llm-complete-guide/pipelines/finetune_embeddings_legacy.py:60]: still wanna keep this one in?

**Review by AlexejPenner — APPROVED** [review]
:seal:

File: llm-complete-guide/pipelines/finetune_embeddings_legacy.py (hunk 2 above)
Comment (strickvl) [reply llm-complete-guide/pipelines/finetune_embeddings_legacy.py:60]: yeah it's part of a bigger experiment. It's in the legacy pipeline. I'd leave it for now.

**Pull request merged**